roles/manage-aws-infra/tasks/manage-cf-stack.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Ensure the underlaying infrastructure is {{ action }}
+  cloudformation:
+    stack_name: "{{ env_id }}"
+    state: "{{ action }}"
+    region: "{{ aws_region }}"
+    template: "{{ cftemplate.dest }}"
```

Comment:
Really no  need to keep this as a file, I'd recommend replacing this with:
```
template_body: "{{ lookup('template', 'casl-aws-cformation.yml.j2') }}"
```

... and then remove the import_tasks and hence the `process-cf-template.yml` file.

Reply:
Done